bridge/bridgeapp/forms.py

```diff
@@ -1,38 +1,49 @@
 from django import forms
 from bridgeapp.models import Category
 
+# global variable with assigned loop to create a dictionary of category.id : category.type key:value pairs
 CAT_CHOICES = [(f"{category.id}", f"{category.type}")
                for category in Category.objects.all()]
 
 
 class ResponseForm(forms.Form):
     """Form used for Responses to Threads"""
+    # simple form with a single CharField
     body = forms.CharField(
         required=True,
         label='',
         widget=forms.Textarea(attrs={
             'rows': '3',
             'cols': '50',
+            # causes cursor to focus on text input area upon page load
             'autofocus': True,
+            # inserts placeholder text into text input area as prompt for user
             'placeholder': 'Write a response here...'
         }))
 
 
 class ThreadForm(forms.Form):
     """Form used for creation of Threads"""
```

Comment:
The form is **not** use to create threads.
It's used for creating **questions** that might lead their respective threads.
Each thread contain responses created by another form.